Create ui-c8y-1021-51-1-fix-filtered-items-count-on-child-devices-view-8430-graft-release.md (#3072)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-51-1-fix-filtered-items-count-on-child-devices-view-8430-graft-release.md

---
date: ""
title: Fixed incorrect filtered items count on child devices view
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM

>     label: Device Management app

Reviewer: Is this supposed to be Device Management? The issue info points to a change in Cockpit.

Reviewer 2: @l2p- yes, it was a bug in Device Management

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62872
version: 1021.51.1
---

> Fix filteredItemsCount on child devices view (#8430) [GRAFT][release/cd] (#8500)

Reviewer 2: ```suggestion
The number of filtered items was displayed as literal text instead of being replaced with the actual value in a child device view. This now works correctly.
```

> The number of filtered items was displayed as literal text instead of being replaced with the actual value in a child device view. This now works correctly.

Reviewer 3: ```suggestion
Previously, the number of filtered items in the child device view was displayed as text instead of being replaced with the actual value. This issue has been fixed and the actual value is displayed now properly. 
```

> Previously, the number of filtered items in the child device view was displayed as text instead of being replaced with the actual value. This issue has been fixed and the actual value is displayed now properly. 

Reviewer 3: ```suggestion
Previously, the number of filtered items in the child devices view was displayed as text instead of being replaced with the actual value. This issue has been fixed and the actual value is displayed now properly. 
```
